src/client/components/Competitors.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+
+import "./styling/competitors.css";
+
+const Competitors = () => {
```

Comment:
change line to 

<code>export const Competitors = () => {</code>

to make it consistent with other components